### ISSUE TITLE
Provider Specialty is always `GENERAL PRACTICE`

### ISSUE THREAD

**Issue by shusson** [synthetichealth/synthea#488]
I generated 100k patients with CSV output and default settings. I noticed that I ended up with 108 providers but all of them had the specialty `GENERAL PRACTICE`. Should this be the case? I'm hoping it would reflect the specialism of the practitioner, e.g Cardiologist. Although I can imagine providers having multiple specialties.

**Comment by jawalonoski** [comment]
Yes, that is expected because we have not modeled specialists yet.

**Comment by jawalonoski** [comment]
This is related to #487

**Comment by shusson** [comment]
Thanks for the info. I think if #487 were to be implemented we could use that instead.

**Comment by shawrk3387** [comment]
I am not able to generate the dataset. Can someone please share the dataset in CSV format if you have already generated one?

**Comment by jawalonoski** [comment]
@shawrk3387 You can download some older/stale pregenerated data from here: https://syntheticmass.mitre.org/download.html

However, if you want fresh (better) CSV data, I suggest [you follow the instructions](https://github.com/synthetichealth/synthea/wiki/CSV-File-Data-Dictionary) and generate data using the latest code.
